src/components/eventlist.js

```diff
@@ -1,16 +1,56 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+    // eslint-disable-next-line no-unused-vars
+    withRouter, BrowserRouter as Router, Route, NavLink, Switch,
+} from 'react-router-dom';
+import { fetchUserEvents } from '../actions';
 import Error from './error';
 
 // Displays list of events user is a part of
-const EventList = (props) => {
-    return (
-        <div className="new-content">
-            <Error />
-            Events for {props.user.username}!
-        </div>
-    );
-};
+class EventList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            eventList: [],
+        };
+    }
+
+    componentDidMount() {
+        // Get list of events for current user
+        this.props.fetchUserEvents(this.props.user.id, (eventList) => { this.setState({ eventList }); });
+    }
+
+    // Retrieves events for current person8
+    getEventsList = () => {
+        return (
+            <ul>
+                {this.state.eventList.map((item) => {
+                    return (
+                        <li key={item.EventID}>
+                            {/* eslint-disable-next-line max-len */}
+                            <NavLink to={`/eventinfo/${item.EventID},${item.EventName},${item.EventTime},${item.EventDescription},${item.IndustryID},${1}`} exact>{item.EventName}, {item.EventTime}</NavLink>
```

Comment:
Use a button not a navlink

Reply:
Why?

Reply:
You right, nvm
Tho maybe the events page should automatically give you my events and then you navigate from there?
Idk this isn't important for not, ignore me

Reply:
That's a good idea! I agree that it could be good to start from the my events thing, i can work on that in the next pr